scripts/take_photo.py

```diff
@@ -1,3 +1,43 @@
-import pi_camera
+#!/usr/bin/env python3
 
-pi_camera.take_photo([0, 0])
+import argparse
+import time
+
+import numpy as np
+
+import context
+import roboplot.config as config
+import roboplot.core.curves as curves
+import roboplot.core.hardware as hardware
+from roboplot.core.gpio.gpio_wrapper import GPIO
+from roboplot.core.camera.camera_wrapper import Camera
```

Comment:
At least on this branch, this should be
`from roboplot.core.Camera.camera_wrapper import Camera `
(i.e. capital 'C').
But I know you've changed it elsewhere.